coordinator/src/main/scala/filodb.coordinator/NodeCoordinatorActor.scala

```diff
@@ -173,11 +186,10 @@ private[filodb] final class NodeCoordinatorActor(metaStore: MetaStore,
   def queryHandlers: Receive = LoggingReceive {
     case q: QueryCommand =>
       val originator = sender()
-      withQueryActor(originator, q.dataset) { _.tell(q, originator) }
+      tempQueryRouter.withQueryActor(originator, q.dataset) { _.forward(q) }
```

Comment:
Can we reject the message if the q.submitTime is more than 5 seconds delayed ?
This way we can stop the indefinite forward.

Reply:
Done in 7eec2ee